docs/UserGuide.md

```diff
@@ -67,205 +72,226 @@ and efficient Graphical User Interface interaction.
+* `TYPE` specified should be either `s` for supplier or `w` for warehouse.
 
 Examples:
 
-* `edit ct/s i/1 n/Alice p/85236417 r/Largest Supplier e/alicekoh@example.com` Edits the first supplier in the displayed supplier list to be named _Alice_ with contact number of _85236417_ and remark of _Largest Supplier_. In addition, email field of supplier is to be edited as _alicekoh@example.com_.
-* `edit ct/w i/2 n/Bob p/67851234 r/Largest Warehouse addr/Jurong Street 11` Edits the second warehouse in the displayed warehouse list to be named _Bob_ with contact number of _67851234_ and remark of _Largest Warehouse_. In addition, address field of warehouse is to be edited as _Jurong Street 11_.
+* `edit ct/s i/1 n/Alice p/85236417 r/Largest supplier e/alicekoh@example.com` : Edits the name, phone, remark and email
+  of the first supplier in the list of suppliers to be `Alice`, `85236417`, `Largest supplier` and
+  `alicekoh@example.com`.
+* `edit ct/w i/2 n/Bob p/67851234 r/Largest warehouse addr/Jurong Street 11` : Edits the name, phone, remark and
+  address of the second warehouse in the list of warehouses to be `Bob`, `67851234`, `Largest warehouse`
+  and `Jurong Street 11`.
 
-### Finding medical product associated with warehouses / suppliers: `find`
+### Finding relevant supplier(s) or warehouse(s): `find`
 
-Finds all suppliers or warehouses whose name, remark and/or products sold/stored matches the provided argument keywords.
+Finds all supplier(s) or warehouse(s) whose name, remark and/or products sold/stored matches the provided keywords.
 
 Format: `find ct/TYPE [n/NAME…​] [pd/PRODUCT_NAME…​] [r/REMARK…​]`
 
-* Keywords specified are case-insensitive.
-* The TYPE specified should be one of these values: s/w.
-* Any combination of the name, product and remark prefix can be provided but at least one of the prefix for name,
-product or remark must be specified.
+* `NAME`, `PRODUCT_NAME` and `REMARK` are case-insensitive.
+* Note that only full words will be matched. `needle` will match `needle` but not `needles`. 
+* `TYPE` specified should be either `s` for supplier or `w` for warehouse.
+* Any combination of the `NAME`, `PRODUCT_NAME` and `REMARK` parameters can be provided but at least one of the
+  parameters with its corresponding prefix must be specified.
 
 Examples:
 
-* `find ct/s pd/masks` Displays all the suppliers that sell masks.
-* `find ct/w pd/PANADOL SUSP r/biggest` Displays the biggest warehouse that stores products matching either PANADOL or
-SUSP.
+* `find ct/s pd/masks` : Displays all the suppliers that sell `masks`.
+* `find ct/w pd/panadol r/biggest` : Displays the warehouse(s) that stores products with names matching
+  `panadol` or with remark matching `biggest`.
 
-### List all suppliers and warehouses entries : `list`
+### Listing all suppliers and warehouses entries : `list`
 
-List all entries (Suppliers and Warehouses) from the CLI-nic.
+List all suppliers and warehouses entries in CLI-nic.
```

Comment:
Based on the language for the other commands, i think this should be "lists". Also maybe there should be an apostophe for "warehouses' entries"